pkgs/by-name/pr/promptfoo/package.nix

```diff
@@ -27,4 +25,13 @@ buildNpmPackage rec {
     license = licenses.mit;
     maintainers = [ maintainers.nathanielbrough ];
   };
+
+   # Prevent Playwright download as that will cause the package build to fail
+  PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD = "1";
```

Comment:
i guess this is functionally equivalent to `env.PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD = "1";` but i'd be nice to make that explicit for the sake of consistency since that's the form that's used elsewhere[1][2][3].

[1] https://github.com/NixOS/nixpkgs/blob/a8d832a56361df7fdeaa9774bceae8745d6403cd/pkgs/by-name/ma/marktext/package.nix#L125-L128
[2] https://github.com/NixOS/nixpkgs/blob/a8d832a56361df7fdeaa9774bceae8745d6403cd/pkgs/by-name/fi/fider/frontend.nix#L34-L36
[3] https://github.com/NixOS/nixpkgs/blob/a8d832a56361df7fdeaa9774bceae8745d6403cd/pkgs/by-name/an/antares/package.nix#L58-L59